app/helpers/work_packages_helper.rb

```diff
@@ -327,22 +327,33 @@ def work_package_show_attributes(work_package)
     group_work_package_attributes work_package_show_attribute_list(work_package)
   end
 
-  def work_package_show_table_row(attribute, klass = nil, attribute_lang = nil, value_lang = nil, &block)
```

Comment:
@NobodysNightmare should be fixed in 3ff4a1a. I guess I ought to go through all the plugins :frowning: